test/complex.cpp

```diff
@@ -43,12 +43,12 @@ constexpr auto test(complex<int> lhs, complex<long long> rhs)
 int main()
 {
     // Constructor tests
-    constexpr imaginary<int> i1;
+    constexpr imaginary<int> i1{};
```

Comment:
I saw that, would you want the other changes in this commit? Macros are ugly, but at the same time.. it's written so many times.